### PR TITLE
Adding support for location measurements

### PR DESCRIPTION
**Pull request by peolivei2**

### CODE REVIEW

File: IoTCIntegration/lib/engine.js [hunk 1]
```diff
@@ -75,14 +75,29 @@ function validateMeasurements(measurements) {
...
 
+/**
+ * @returns true if a measurement is a location.
+ */
+function isLocation(measurement) {
+    if (typeof measurement !== 'object' || typeof measurement.lat !== 'number' || typeof measurement.lon !== 'number') {
+        return false;
+    }
+
+    if ('alt' in measurement && typeof measurement.alt !== 'number') {
```
Comment (princjef) [inline IoTCIntegration/lib/engine.js]: not sure how strict we want the validation of locations to be, but do we care if they have extra fields in the object other than `lat`, `lon` and `alt`?

File: IoTCIntegration/lib/engine.js [hunk 2]
```diff
@@ -75,14 +75,29 @@ function validateMeasurements(measurements) {
...
     }
 
     return true;
 }
 
+/**
+ * @returns true if a measurement is a location.
+ */
+function isLocation(measurement) {
+    if (typeof measurement !== 'object' || typeof measurement.lat !== 'number' || typeof measurement.lon !== 'number') {
```
Comment (princjef) [inline IoTCIntegration/lib/engine.js]: might want to have a `measurement &&` in the front here since `typeof null === 'object'`

Comment (peolivei2) [reply IoTCIntegration/lib/engine.js]: Good catch. I really didn't know that :)

File: IoTCIntegration/lib/engine.js (hunk 1 above)
Comment (peolivei2) [reply IoTCIntegration/lib/engine.js]: We don't care if there are extra fields. It's fine if users have a more complex location object in their side.

**Pull request merged**